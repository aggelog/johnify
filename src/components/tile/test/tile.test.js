import React from 'react';
import Tile from '../index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Tile title="hello world" gif={{}} id="1" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});