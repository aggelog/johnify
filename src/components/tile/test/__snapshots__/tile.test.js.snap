// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="col-xs-12 col-md-6 col-lg-4 col-xl-3"
>
  <div
    className="tile__container tile__shadow"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="tile-header__container"
    >
      <div
        className="title-header__title"
      >
        hello world
      </div>
    </div>
    <div
      className="tile-media__container"
    >
      <div
        className="tile-media__item-inside"
      >
        <div
          className="tile-media__loading-container"
        >
          <div>
            <span
              className="l-1"
            />
            <span
              className="l-2"
            />
            <span
              className="l-3"
            />
            <span
              className="l-4"
            />
            <span
              className="l-5"
            />
            <span
              className="l-6"
            />
          </div>
        </div>
        <img
          alt="..."
          className="tile-media__image tile-media__image-loading"
          onLoad={[Function]}
          src={undefined}
        />
      </div>
    </div>
  </div>
</div>
`;
